docs/concepts/actions.md

```diff
@@ -0,0 +1,64 @@
+# Actions
+
+Actions are _the_ way to change the state. Actions have this signature: `payload => (state, actions) => newState`. Actions are passed to the `app` as the second parameter ([app](tbd)).
+
+## Usage
+
```

Comment:
These explanation is too elaborate.

Reply:
Actions are _the_ way to change the state and have this signature: `payload => (state, actions) => newState`.

***

That's nice and short!